twoliter/src/project.rs

```diff
@@ -56,6 +59,10 @@ pub(crate) struct Project {
 
     /// Set of kit dependencies
     kit: Vec<Image>,
+
+    /// Set of overrides read from Twoliter.override
+    #[serde(skip)]
+    overrides: BTreeMap<String, BTreeMap<String, Override>>,
```

Comment:
Since we have `UnvalidatedProject`, should we just remove the `serde` tag from this and do away with `serde(skip)`?